migrations/20240403201102_init_tables.sql

```diff
@@ -1,15 +1,16 @@
 -- +goose Up
 -- +goose StatementBegin
-CREATE TABLE users (
+CREATE TABLE IF NOT EXISTS users (
   id SERIAL PRIMARY KEY,
   name VARCHAR(255) NOT NULL,
   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
 
-INSERT INTO users (name) VALUES ('Alice');
+INSERT INTO users (created_at, updated_at, email, username, password)
+VALUES (NOW(), NOW(), 'statuspage@api.com', 'StatusPageAPI', 'b63729771dbc8166067f73c735c7a78dde61ccbe25d05d0cefec4ba2acbcfa23');
```

Comment:
Something is wrong here, in the above statement of creating the users table there is no updated_at, email, username, password fields, how this works then?
And I would suggest adding the created, updated dates at the end.